corehq/apps/app_manager/views/modules.py

```diff
@@ -1114,8 +1114,14 @@ def edit_module_detail_screens(request, domain, app_id, module_unique_id):
         ):
             search_label = module.search_config.search_label
             search_label.label[lang] = search_properties.get('search_label', '')
+            search_label._set_media("media_image", lang, search_properties.get('search_label_image'))
```

Comment:
this method is called at another place already, considering making it public (removing the `_`).

Reply:
I agree with removing the `_`.

Reply:
resolved by https://github.com/dimagi/commcare-hq/pull/29765/commits/89838899561e2de0f4a306ae4392c5ed8f1944e2